src/Common/src/System/IO/StringBuilderCache.cs

```diff
@@ -5,7 +5,6 @@
 
 /*============================================================
 **
-** Class:  StringBuilderCache
```

Comment:
It must have been one of the auto-fixes we applied on the projects. Will revert.